src/NewRecipe.css

```diff
@@ -0,0 +1,65 @@
+.New {
+  margin-bottom: 50px auto;
+  margin-top: 0px auto;
+  background-color: #282c34;
+  min-height: 100vh;
+  clear: both;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.New-Body {
```

Comment:
the css class names leave a bit to be desired, there's lots of different ways to go about naming. I'd look into notation such as .new body {
} if you mean any body classes under new or 
.new > body {
}
If you mean a direct body element under new